code-challenges.js

```diff
@@ -31,38 +44,77 @@ const temp = 42
 // const temp = 212
 // Expected output: "212 is at boiling point"
 
-// --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
+ //--------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
-// Describe your process:
+// Describe your process: 1.I created two variables for the two different arrays
+//2.I console log with the concat method to combine the arrays
+//3.I console log the first variable since they are combined and used the length method to show how many indexes their are in the array.
 
-const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
-const padres1998WorldSeriesRuns = [6, 3, 5, 3]
-// Expected output: 9
+let padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+let padres1998WorldSeriesRuns = [6, 3, 5, 3]
+console.log(padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns))
+console.log(padres1998WorldSeriesRuns.length)
+//const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+//const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+ //Expected output: 9
 
-// --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
-// Describe your process:
+ 
+ // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
+
+ //1.created a variable named currentCohort
+ //2.declared the variable as an array foxtrot 2022
+ //3.console.log the variable with the reverse 
+
+let currentCohort = ["Foxtrot 2022"]
+ console.log(currentCohort.reverse)
```

Comment:
Make sure to use the given variables.  Your variable differs from the initial variable given and will change your approach to this problem a bit.